news/_posts/core-weekly/2021-01-12-coreweekly-02-2021.md

```diff
@@ -0,0 +1,232 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 2 of 2021"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2021-01-12
+authors: [ PrestaShop ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+twitter_image: /assets/images/theme/banner-core-weekly.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 4th to Sunday 10th of January 2021.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+Dear Developers,
+
+PrestaShop [1.7.7.1](https://build.prestashop.com/news/prestashop-1-7-7-1-maintenance-release/) has been released! don't forget to update as this patch version fixes 41 regressions reported on version 1.7.7.0.
+
+The kanban for `1.7.7` branch is not empty yet, it still contains unresolved regressions reported on version 1.7.7.0. The next delivery of bugfixes for 1.7.7 will be delivered in patch version 1.7.7.2.
```

Comment:
```suggestion
The kanban for `1.7.7` branch is not empty yet, it still contains unresolved regressions reported on version 1.7.7.0. The next delivery of bugfixes for 1.7.7 will be delivered in patch version 1.7.7.2 in six weeks.
```

Reply:
I'm not sure about this. We should check with 1.7.7.2 product manager if 1.7.7.2 is already approved and scheduled?

Also 6 weeks is a maximum duration 😉  we can deliver sooner if needed

Reply:
then, "...in maximum six weeks."

Reply:
Yes, it is already approved. For schedule, 6 weeks since last release (so yesterday)

Reply:
Already approved, I'm still working on an issue for the 1.7.7.2 and we have to fix autoupgrade problems :)

Reply:
OK. I then create the tracker issue https://github.com/PrestaShop/PrestaShop/issues/22783

Reply:
I change my suggestion above.

Reply:
Already did